### ISSUE TITLE
BIG LOSS

### ISSUE THREAD

**Issue by tangbohu** [jwyang/faster-rcnn.pytorch#628]
It seems that the loss keeps around 0.8 during training. It is right?

**Comment by zyyll** [comment]
My loss is 0.4~0.5 in pascal2007 and vgg16

**Comment by devendraswamy** [comment]
how can i reduce the loss , any idea please help me
